dashboards/vmagent.json

```diff
@@ -1,5 +1,14 @@
 {
-  "__inputs": [],
+  "__inputs": [
+    {
+      "name": "DS_VICTORIAMETRICS",
+      "label": "VictoriaMetrics",
+      "description": "",
+      "type": "datasource",
+      "pluginId": "prometheus",
+      "pluginName": "Prometheus"
+    }
+  ],
```

Comment:
This change shouldn't be here if you export dashboard with "sharing externally" button:
<img width="384" alt="image" src="https://github.com/VictoriaMetrics/VictoriaMetrics/assets/2902918/2b06709d-4712-4b7b-b989-6947b05739ed">

Othwerwise, it breaks this dashboard. You can verify it by restarting the docker-compose with ` make docker-single-down` and then ` make docker-single-up`